src/FormsyContext.ts

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { FormsyContextInterface } from './interfaces';
+
+const noFormsyErrorMessage = 'Could not find Formsy Context Provider. Did you use withFormsy outside Formsy ?';
+
+const throwNoFormsyProvider = () => {
+  throw new Error(noFormsyErrorMessage);
```

Comment:
Will this stop formsy components from being able to be used outside of Formsy without throwing errors?

This may be too harsh a response. That said, I do like that it's very clear to users who are making a mistake and I normally really appreciate helpful errors like this.

I guess you could always export your input components twice, once withFormsy and once without.

Reply:
Yes it will crash the application if not catched. The same behavior as react-redux if you call connect outside a Provider.
If you think it should only print a error in the console let me know.